imdb.py

```diff
@@ -4,56 +4,80 @@
 import json
 import bs4
 
+#clears the screen in python
+os.system('clear')			
 
-os.system('clear')
-
-
-status=open('info.txt','a')
+#creates/opens info.tx in open mode
+status=open('info.txt','a')		
 
 
 
 def info_movie():
-    name = raw_input('\nEnter the title of the movie: ')
-    t = name.replace(' ','%20')
+	
+    name = raw_input("\n Enter the title of the movie: ")
+	
+	#replaces all whitespaces with string '%20'
```

Comment:
The indentation of documentation and commands a-is not same. Also, leave a space after `#` and write the comments in Sentence Case.

Reply:
but in my local file , the indentation is perfectly fine ( i checked on vscode and notepad++). How does it changes here. And likewise is there any biasness of one editors's indentation over other?

Reply:
@akshatnitd